tests/driver_mrf24j40/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = driver_mrf24j40
```

Comment:
It is basically the same driver test as the `driver_at86rf2xx` test (so, not much effort). Since (I believe) the `gnrc_networking` example also has a test on the link layer (the function called `txtsnd`)  I agree.